orchagent/port.h

```diff
@@ -11,7 +11,7 @@ extern "C" {
 #include <map>
 
 #define DEFAULT_PORT_VLAN_ID    1
-#define DEFAULT_MTU              9100
+#define DEFAULT_MTU             1492
```

Comment:
>1492 [](start = 32, length = 4)

Is it derived from SAI_PORT_ATTR_MTU? Can we use a const expression here? Or add some comment.

Reply:
Prefer to keep it as before. Added comment for clarify.